### DISCUSSION TITLE
How to get github achievements?

### DISCUSSION THREAD

**Discussion by vfernando01** [community/community#62517] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to get github achievements?

**Comment by franciscusvalentinus** [comment]
As of my last update in September 2021, GitHub does not have a formal "achievements" system like some gaming platforms. However, GitHub does provide certain features and recognition that developers can earn and showcase as accomplishments. Here are some ways to gain recognition on GitHub:

1. **Contributions**: Make meaningful contributions to open-source projects or repositories. Contributions can include code changes, bug fixes, documentation improvements, or creating new features.

2. **Pull Requests and Reviews**: Submitting pull requests and participating in code reviews can show your involvement in collaborative development.

3. **Stars**: When others find your repository valuable, they may "star" it, indicating their appreciation for your work.

4. **Forks**: If developers find your repository useful, they might fork it to use as a starting point for their own projects.

5. **Followers**: Gain followers who are interested in your work or projects.

6. **GitHub Profile README**: You can customize your GitHub profile README to showcase your skills, projects, or anything you want to highlight.

7. **GitHub Badges**: Some developers use badges in their repositories' README to show information like build status, code coverage, and more.

8. **GitHub Contributions Graph**: The contributions graph on your GitHub profile shows your activity over time, indicating your level of involvement.

Remember that GitHub is a platform for collaboration and sharing your work with the community. The focus should be on contributing positively, creating valuable content, and building connections with other developers.

Keep in mind that GitHub's features and community may have evolved since my last update in September 2021, so it's a good idea to check GitHub's official documentation and community guidelines for the latest information and best practices.

**Comment by TRZMlEL** [comment]
in this video u have full instructions https://www.youtube.com/watch?v=oJWFfp3sGYA&t=251s

**Comment by queenofcorgis** [comment, marked as answer]
@vfernando01 @franciscusvalentinus 

GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
